src/main/java/frc/robot/Robot.java

```diff
@@ -27,7 +25,6 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
-    m_robotContainer = new RobotContainer();
```

Comment:
Looks like we lost robot container somehow. we should revert back to original Robot.java